marshal/message.go

```diff
@@ -233,3 +248,26 @@ func (m *msgClaimingMessages) Type() msgType {
 func (m *msgClaimingMessages) Timestamp() int {
 	return m.Time
 }
+
+// msgReleaseGroup is used by the Admin to pause a consumer group,
+// identified by groupID, until MsgExpireTime.
+type msgReleaseGroup struct {
+	msgBase
+	MsgExpireTime time.Time
+}
+
+// Encode returns a string representation of the message.
+func (m *msgReleaseGroup) Encode() string {
+	return "ReleaseGroup/" + m.msgBase.Encode() +
+		fmt.Sprintf("/%s", m.MsgExpireTime.Format(time.UnixDate))
```

Comment:
Er, this puts a non-integer in? This field should really be an integer shouldn't it?